packages/cli/src/config/app.rs

```diff
@@ -8,8 +8,15 @@ pub(crate) struct ApplicationConfig {
 
     #[serde(default)]
     pub(crate) sub_package: Option<String>,
+
+    #[serde(default = "out_dir_default")]
+    pub(crate) out_dir: PathBuf,
 }
 
 pub(crate) fn asset_dir_default() -> PathBuf {
     PathBuf::from("assets")
 }
+
+pub(crate) fn out_dir_default() -> PathBuf {
+    PathBuf::from("dist")  
```

Comment:
It may break the previouse deployment script due to changing default build artifact directory. How about returning PathBuf for `target/dx`, which is current out directory in `dioxus_crate.rs:100`